### DISCUSSION TITLE
Provide firmware for the nouveau project

### DISCUSSION THREAD

**Discussion by pavlokozlenko** [NVIDIA/open-gpu-kernel-modules#603] [Ideas]
Hi Why not provide the necessary firmware (for power management) to the nouveau project? I'm talking about the GM20x, GP10x and GV100 architectures. It is the GP10x and GV100 that suffer from a lack of power management. For you, this will raise karma and more and more people will want to buy nvidia products. Please provide firmware for nouveau project. You have already provided the GSP firmware for which we are very grateful to you, it gave a breath for nouveau. Why not do this for older architectures?

**Comment by pavlokozlenko** [comment]
@aritger

**Comment by pavlokozlenko** [comment]
Is NVIDIA\@aritger afraid to answer or what? Tell me straight - we will never provide you with firmware. I wanted to collect money a graphics card  TU1**, but I see that you just don't care about users. My next graphics card will be amd.

**Reply by aritger** [reply]
Hi @PavloIT115.  I'm sorry; we currently have no plans to provide ucode for Nouveau for pre-Turing GPUs.

**Reply by pavlokozlenko** [reply]
Why? Society has been begging you to do this for many years... Is it really that difficult?

**Comment by aritger** [comment]
I'm sorry; it wasn't my decision, and I don't believe there is anything else constructive we can say about it here.

For better or worse, our focusing going forward will be to enable use of NVIDIA's GSP firmware.

**Reply by pavlokozlenko** [reply]
@aritger Please excuse me for my rudeness. I shouldn't have written messages with such a rude tone

**Comment by Ramen-LadyHKG** [comment]
I'm using `GTX 1060 Mobile 6GB` on `Microsoft Surface Book 2`

The GPU fan does not turn on until 70C degree, is nvidia driver causing this issue?

[https://forums.developer.nvidia.com/t/changing-power-management-limit-is-not-supported-for-gpu-pascal-gtx1060-laptop-mobile-linux-555-nvidia-smi-powerlimit/298873](url)

Why removed power management support in 535 version when it was supported in 525 version?###
